src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -64,13 +85,15 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges {
   search = '';
   rows = [];
   selected = [];
-  loadingIndicator = false;
+  loadingIndicator = true;
   paginationClasses = {
     pagerLeftArrow: 'i fa fa-angle-double-left',
     pagerRightArrow: 'i fa fa-angle-double-right',
     pagerPrevious: 'i fa fa-angle-left',
     pagerNext: 'i fa fa-angle-right'
   };
+  private subscriber;
+  private updating = false;
```

Comment:
This variable should be removed. You can use `loadingIndicator `to store the same information.
Or you can remove `loadingIndicator `and keep `updating`.

Reply:
They both serve different things, `loadingIndicator` is something for the user to observe, `updating` is for making sure the last request was a success. In a view minutes (not pushed yet) you will see that the first variable will be set to true again if the user hits refresh.

Reply:
done

Reply:
`loadingIndicator  `and `updating `will always have the same value. Please remove one.

Reply:
They don't have the same value only on the first fetch and if you manually hit the refresh button. As described there is a difference.